src/org/zaproxy/zap/extension/bugtracker/DialogAddBugzillaConfig.java

```diff
@@ -134,7 +134,7 @@ protected boolean validateFields() {
     
     @Override
     protected void performAction() {
-        config = new BugTrackerBugzillaConfigParams(getNameTextField().getText(), getPasswordTextField().getText(), getBugzillaUrlTextField().getText());
+        config = new BugTrackerBugzillaConfigParams(getNameTextField().getText(), getPasswordTextField().getPassword().toString(), getBugzillaUrlTextField().getText());
```

Comment:
This change does not yield the same result, it needs to created a new String but better resolution of the deprecation is to use the char array.